adafruit_mpl3115a2.py

```diff
@@ -278,13 +278,13 @@ def temperature(self):
     def sealevel_pressure(self):
         """Read and write the pressure at sea-level used to calculate altitude.
         You must look this up from a local weather or meteorological report for
-        the best accuracy.  This is a value in Pascals.
+        the best accuracy.  This is a value in Hectopascals.
         """
         # Read the sea level pressure in bars.
         self._read_into(_MPL3115A2_BAR_IN_MSB, self._BUFFER, count=2)
-        # Reconstruct 16-bit value and scale back to pascals.
+        # Reconstruct 16-bit value and scale back to Hectopascals.
         pressure = (self._BUFFER[0] << 8) | self._BUFFER[1]
-        return pressure * 2.0
+        return pressure * 2.0 / 100
 
     @sealevel_pressure.setter
     def sealevel_pressure(self, val):
```

Comment:
The setter for `sealevel_pressure` should also be changed to accept Hectopascals since the getter is being modified.

Reply:
please refer to https://github.com/adafruit/Adafruit_CircuitPython_MPL3115A2/pull/27/commits/a416817688da335c8b5f1aa45a50990e087094d0.

Thanks :)